node/common/polyfill/text_encoder.ts

```diff
@@ -11,10 +9,48 @@ export class TextEncoder {
    * @return {Uint8Array}
    */
   encode(encoding: string): Uint8Array {
-    return !encoding
-      ? new Uint8Array()
-      : new Uint8Array(
-          jsrsasign.hextoArrayBuffer(jsrsasign.utf8tohex(encoding))
-        );
+    const hex = encodeURIComponentAll(encoding)
+      .replace(/%/g, '')
+      .toLocaleLowerCase();
+    const buffer = new ArrayBuffer(hex.length / 2);
+    const view = new DataView(buffer);
+
+    for (let i = 0; i < hex.length / 2; i++) {
+      view.setUint8(i, parseInt(hex.substring(i * 2, i * 2 + 2), 16));
+    }
+
+    return new Uint8Array(buffer);
   }
 }
+
+function encodeURIComponentAll(u8: string) {
+  const s = encodeURIComponent(u8);
+  let s2 = '';
+  for (let i = 0; i < s.length; i++) {
+    if (s[i] === '%') {
+      s2 = s2 + s.substring(i, i + 3);
+      i = i + 2;
+    } else {
+      // A-Z a-z 0-9 - _ . ! ~ * ' ( )
+      s2 = s2 + '%' + stohex(s[i]);
+    }
+  }
+  return s2;
+}
+
+function stohex(s: string) {
```

Comment:
Not sure, but jsrsasign seems to have the same-named API. Did you port this from jsrsasign? Since the license of jsrsasign is MIT, I think we can use it, but we need to show the license and copyright. If you write this from scratch, forget about this. Thanks.
https://kjur.github.io/jsrsasign/api/symbols/global__.html#encodeURIComponentAll

Reply:
@jnmt 
Jun, thank you for pointing that out.

Yes.
This part is copied from jsrsasign package (I shortened it a bit only).

Sorry, I miss the license part.

According to MIT license, we have to mention the author and the license.

Let me put them in the comment.

Reply:
@jnmt 

I made the description in https://github.com/scalar-labs/scalardl-javascript-client-sdk/pull/10/commits/23c7bb22bcd37c4aadc2990ede41227a34e55f77

Please take a look!